### DISCUSSION TITLE
Assets don’t load with nginx reverse proxy

### DISCUSSION THREAD

**Discussion by nevetS-718** [YOURLS/YOURLS#3056] [Q&A] [answered]
I'm using nginx only as a reverse proxy and have a web server running html and php sites on the backend. When I route my YOURLS site through nginx, it loses all of the assets, but still works. The site is https://url.nevets.tech and I am unable to access the admin interface with a 404 error.

**Comment by dgw** [comment, marked as answer]
Nginx is serving the page over HTTPS, but YOURLS doesn't know this and embeds HTTP scripts/styles, which are then blocked as [mixed content](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content). Tell YOURLS that it's being served securely. Set [`YOURLS_SITE`](https://yourls.org/#Config;site) to `https://url.nevets.tech`.

**Reply by nevetS-718** [reply]
it works! thank you!
